src/resolvers/OrganizationResolver.ts

```diff
@@ -45,17 +53,17 @@ class OrganizationResolver {
   }
 
   @FieldResolver((_returns) => Number, {
-    description: "The total shoutouts of an <Organization's> <Flyers>",
+    description: 'The total number of <Flyers> from an <Organization>',
```

Comment:
nit: keep description consistent as other ones (e.g. "Returns the total # ...")

Reply:
prod sg